__init__.py

```diff
@@ -443,13 +429,16 @@ def remove_function_annotations(self, src: str) -> str:
         rest is kept the same.
         """
         tree = self.parser.parse(bytes(src, "utf8"))
+        # captures now returns a dict of capture name -> list of nodes
```

Comment:
This comment probably isn't needed. Just delete, and we'll pin in requirements.txt